### DISCUSSION TITLE
zh/docs/master/quick-start

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#96] [Giscus]
# zh/docs/master/quick-start

快速上手 本指南将引导您基于GitRec的数据集以最快的速度为 GitHub 仓库搭建一个推荐系统。在开始前，请确保你已经安装了以下软件： Docker ，Gorse 的节点将运行在容器中。 Docker Compose ，Gorse 的节点由 Docker Compose 编排。

https://gorse.io/zh/docs/master/quick-start.html

**Comment by shenhuanjie** [comment]
受益匪浅

**Comment by ghost** [comment]
好奇怪, 为啥一定要Docker, go的优势就是单文件好部署. 你这额外多个Docker要求, 不是设置了入门门槛?

**Reply by ghost** [reply]
好吧, 还有二进制部署, ...

**Comment by qianduixue** [comment]
为啥二进制部署 老是要我提示 /lib/lib.so.6 GLIBC_2.34 NOT Found
